connector_magento/models/product/importer.py

```diff
@@ -196,6 +195,18 @@ class ProductImportMapper(Component):
               (normalize_datetime('updated_at'), 'updated_at'),
               ]
 
+    @mapping
```

Comment:
As far as i know - the weight in odoo is fixed to kg - it does not depend on the product uom. So weight in magento and odoo is in kg - no need for this in my opinion.

Reply:
I think we can leave this change in there - because maybe sometimes magento will change the default weight - and then we are prepared...
Setting the default to kg would be great.